src/camera.rs

```diff
@@ -12,16 +12,14 @@ struct MainCameraBundle {
     #[bundle]
     input_manager: InputManagerBundle<Action>,
     #[bundle]
-    pbr: PerspectiveCameraBundle,
+    perspective_camera: PerspectiveCameraBundle,
```

Comment:
`pbr` stands for "Physically based rendering" in bevy generally, and anyway, it would be pcr :P